### PR TITLE
Upgraded to play 2.7.3 and slick 3.3.2

### PR DESCRIPTION
**Pull request by KacperFKorban**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: scala
 scala:
   - "2.11.12"
-  - "2.12.7"
+  - "2.12.6"
```
Comment (JakubSroka) [inline .travis.yml]: 2.12.8

File: build.sbt [hunk 2]
```diff
@@ -19,13 +19,15 @@ resolvers += Resolver.sonatypeRepo("releases")
 resolvers += Resolver.sonatypeRepo("snapshots")
 
 libraryDependencies ++= Seq(
-  "org.virtuslab" %% "unicorn" % "1.3.0",
-  "org.scalatestplus.play" %% "scalatestplus-play" % "3.0.0" % Test,
-  "com.typesafe.play" %% "play-test" % "2.6.20" % "test",
-  "com.typesafe.play" %% "play-json-joda" % "2.6.10",
-  "com.h2database" % "h2" % "1.4.187" % "test"
+  "org.virtuslab" %% "unicorn" % "1.3.3-SNAPSHOT",
```
Comment (JakubSroka) [inline build.sbt]: remove SNAPSHOT

Comment (bartosz822) [reply build.sbt]: for now it could be snapshot until we release it

File: .travis.yml (hunk 1 above)
Comment (KacperFKorban) [reply .travis.yml]: Done.

**Comment by bartosz822** [comment]
maybe try to rebase this with parallel tests fixes, it should work better on travis

**Comment by JakubSroka** [comment]
old

**Pull request closed without merging**